test/integration/check/test_app/correct_syntax.lua

```diff
@@ -0,0 +1,10 @@
+default_cfg = {
```

Comment:
Let's not use anything that looks like a legal default config for `box.cfg` for the test, it's confusing. It looks like 2-3 lines of meaningless code is enough for the test.

Reply:
Done